youtube_dl/extractor/rte.py

```diff
@@ -60,3 +56,71 @@ def _real_extract(self, url):
             'thumbnail': thumbnail,
             'duration': duration,
         }
+
+
+
+class RteRadioIE(InfoExtractor):
+    IE_NAME = 'rte.ie'
+    IE_DESC = 'Raidió Teilifís Éireann Radio'
+    _VALID_URL = r'''(?x)
+        https?://(?:www\.)?rte\.ie/radio/utils/radioplayer/rteradioweb\.html
+        [#]!rii=(?:[0-9]+)(?:%3A|:)  # Channel ID
```

Comment:
OK. Generally, for documentation purposes, I prefer to split non-trivial regexps into fragments that can be individually commented, but I'm not going to bikeshed over it.

By the way, do you know if there's any way to delete the now-redundant travis-ci jobs #8130 and #8132 before they run?